src/main/scala/com/mwz/sonar/scala/scapegoat/ScapegoatSensor.scala

```diff
@@ -24,6 +24,8 @@ import scala.util.{Failure, Success, Try}
 
 import cats.instances.string._
 import cats.syntax.eq._
+import com.mwz.sonar.scala.metadata.scapegoat.ScapegoatRules
+import com.mwz.sonar.scala.metadata.scapegoat.ScapegoatRulesRepository
```

Comment:
I see this imported but not used.

Reply:
Yeah, these imports are used in this file, e.g. 
here
https://github.com/mwz/sonar-scala/pull/383/files/90d52f74b3f038f0f3b4946fdb035ba41e374f5f#diff-d8c93bc20bbe202f3993e73d6bdf3c53R52
and here
https://github.com/mwz/sonar-scala/pull/383/files/90d52f74b3f038f0f3b4946fdb035ba41e374f5f#diff-d8c93bc20bbe202f3993e73d6bdf3c53R157

You can see `ScapegoatRules` on line 157, but ScapegoatRulesRepository was referencing another package before which I swapped with the `metadata` package, so you'd need to expand the code to see the references.

Reply:
Uhm, well I use glasses for a reason! 🤓